packages/block-library/src/image/index.php

```diff
@@ -27,16 +28,88 @@ function render_block_core_image( $attributes, $content ) {
 		// which now wraps Image Blocks within innerBlocks.
 		// The data-id attribute is added in a core/gallery `render_block_data` hook.
 		$processor->set_attribute( 'data-id', $attributes['data-id'] );
+	}
+
+	$link_destination = isset( $attributes['linkDestination'] ) ? $attributes['linkDestination'] : 'none';
+
+	// Get the lightbox setting from the block attributes.
+	if ( isset( $attributes['behaviors']['lightbox'] ) ) {
+		$lightbox = $attributes['behaviors']['lightbox'];
+		// If the lightbox setting is not set in the block attributes, get it from the theme.json file.
+	} else {
+		$theme_data = WP_Theme_JSON_Resolver_Gutenberg::get_merged_data()->get_data();
+		if ( isset( $theme_data['behaviors']['blocks']['core/image']['lightbox'] ) ) {
+			$lightbox = $theme_data['behaviors']['blocks']['core/image']['lightbox'];
+		} else {
+			$lightbox = false;
+		}
+	}
+
+	$experiments = get_option( 'gutenberg-experiments' );
+
+	if ( ! empty( $experiments['gutenberg-interactivity-api-core-blocks'] ) && 'none' === $link_destination && $lightbox ) {
+
+		$aria_label = 'Open image lightbox';
```

Comment:
Is there a reason for not making this text translatable?

Reply:
It definitely should be translated 👍